newIDE/app/src/ProjectManager/index.js

```diff
@@ -1056,6 +1057,7 @@ export default class ProjectManager extends React.Component<Props, State> {
           value={searchText}
           onRequestSearch={this._onRequestSearch}
           onChange={this._onSearchChange}
+          elevation={24}
```

Comment:
This seems very high no? I don't think there is other Paper as "elevated" in the rest of the codebase. I would rather use a lower number to avoid a shadow to important :)

Reply:
Yes it's the max I guess, either way it doesn't do much. 24 is pretty much similar to 5 :p I'll lower it.